components/login-form.html

```diff
@@ -0,0 +1,41 @@
+<div class="flex justify-center items-center h-screen bg-gray-200 px-6">
+    <div class="p-6 max-w-sm w-full bg-white shadow-md rounded-md">
+        <div class="flex justify-center items-center">
+            <span class="text-gray-700 text-2xl">Login</span>
+        </div>
+
+        <form class="mt-4">
+            <label class="block">
+                <span class="text-gray-700 text-sm">Email</span>
+                <input type="email" class="form-input mt-1 block w-full rounded-md focus:border-indigo-600"
+                />
+            </label>
+
+            <label class="block mt-3">
+                <span class="text-gray-700 text-sm">Password</span>
+                <input type="password" class="form-input mt-1 block w-full rounded-md focus:border-indigo-600"
```

Comment:
Adding an autocomplete for the user's current password, will make the browser use the current password if it's stored. Also, an ID for the label and name, cuz best practices? 🤔 And a `required` since a password should be required.

```suggestion
                <input type="password" id="password" name="password"  autocomplete="current-password" class="form-input mt-1 block w-full rounded-md focus:border-indigo-600" required
```